test/extensions/filters/http/local_ratelimit/filter_test.cc

```diff
@@ -99,25 +100,25 @@ class FilterTest : public testing::Test {
 };
 
 TEST_F(FilterTest, Runtime) {
-  setup(fmt::format(config_yaml, "1", "false"), false, false);
+  setup(fmt::format(config_yaml, "1", "false", "\"OFF\""), false, false);
```

Comment:
I don't think you need to include the quotes in there? You don't when you specify the other option

Reply:
No, we cannot remove these quotes. We write YAML like
```
local_rate_limit_per_downstream_connection: false
enable_x_ratelimit_headers: "OFF"
```
, so we must keep these quotes.